changes/2804.feature.rst

```diff
@@ -0,0 +1 @@
+:py:class:`LocalStore` learned to ``delete_dir``.
```

Comment:
```suggestion
Implemented ``delete_dir`` for :py:class:`LocalStore`. This means it is now possible to delete an entire directory at once, instead of having to delete every individual key under a given directory in the store.
```

Reply:
I'll add another line.